rebasehelper/cli.py

```diff
@@ -159,8 +159,11 @@ def run():
         debug_log_file = None
         try:
             # be verbose until debug_log_file is created
-            handler = LoggerHelper.add_stream_handler(logger, logging.DEBUG, formatter)
-            handler.choose_color_map()
+            main_handler = LoggerHelper.add_stream_handler(logger, logging.DEBUG, formatter)
+            output_tool_handler = LoggerHelper.add_stream_handler(logger_output)
```

Comment:
All these handlers and loggers are a real mess. There should be just different loggers importable from `rebasehelper.logger` with methods for setting log level and color map.